frontend/src/components/views/ProjectileView.jsx

```diff
@@ -27,11 +28,35 @@ const renderLaser = (laser) => {
         .map((point) => {
             const [x, y, z] = point;
             return convertToRenderCoordinates({x, y, z});
-        });
+        })
     return (<Line 
         key={`${spawnedAt},${path[0][0]},${path[0][1]},${path[0][2]}`} 
         points={points} color="red" lineWidth={2} 
-    />);
+    />)
+}
+
+const renderSpikes = (spikes, gltf) => {
+    const { spawnedAt, position, quaternion, scale } = spikes;
+    const ARRAY_OFFSET = 0.2;
+    const spikesArray = [];
+    for (let x = -ARRAY_OFFSET; x <= ARRAY_OFFSET; x+=ARRAY_OFFSET) {
+        for (let y = -ARRAY_OFFSET; y <= ARRAY_OFFSET; y+=ARRAY_OFFSET) {
+            const coordinates = convertToRenderCoordinates(spikes, {x, y: y - 0.02, z: -0.3});
+            spikesArray.push(coordinates);
+        }
+    }
+
+    return (<group key={position}>
+        {spikesArray.map((coordinates) => {
+            return <primitive
+                key={`spikes${spawnedAt},${coordinates.x},${coordinates.y},${coordinates.z}`}
```

Comment:
Is there a way we can make the keys across projectiles a little more consistent? For example, laser doesn't have laser as a prefix.